include/z64.h

```diff
@@ -966,7 +966,7 @@ typedef enum {
     /*  6 */ TRANS_TYPE_FADE_BLACK_SLOW,
     /*  7 */ TRANS_TYPE_FADE_WHITE_SLOW,
     /*  8 */ TRANS_TYPE_WIPE_FAST,
-    /*  9 */ TRANS_TYPE_FILL_WHITE2, 
```

Comment:
Not sure if this got talked about, but based on https://github.com/zeldaret/oot/blob/master/src/code/z_play.c#L554-L559 this looks to have the intention of being `TRANS_TYPE_FILL_WHITE_FAST`, though because of the bug it just acts like a second copy of `TRANS_TYPE_FILL_WHITE`.

Reply:
No that wasn't noticed in the transition PR iirc
You are most likely right, but what do we make of this?

Reply:
Personally I like the idea of renaming it based on this intent, probably with a comment that it doesn't work because of the bug.

Reply:
Why not

I would like this to be done in another PR though, this one already has enough stuff and also reviews (and the waterbox stuff delayed it already)